### DISCUSSION TITLE
Album Art

### DISCUSSION THREAD

**Discussion by rajeshrvds** [Shabinder/SpotiFlyer#57] [Ideas]
Pls increase arbum art quality.

**Comment by Shabinder** [comment]
I decreased it intentionally , to not increase mp3 file size ...

Seems some people want HD album art , I will consider giving an option , so user could choose himself/herself which quality he/she wants.

**Comment by ghost** [comment]
@Shabinder add song preview button on album art before downloading

**Reply by Shabinder** [reply]
What fo you mean by song preview?

**Reply by ghost** [reply]
Yes

**Reply by Shabinder** [reply]
What do you mean by song preview , explain what fo you expect?
